Mathlib/Order/InitialSeg.lean

```diff
@@ -50,12 +56,12 @@ structure InitialSeg {α β : Type*} (r : α → α → Prop) (s : β → β →
   /-- The order embedding is an initial segment -/
   mem_range_of_rel' : ∀ a b, s b (toRelEmbedding a) → b ∈ Set.range toRelEmbedding
 
--- Porting note: Deleted `scoped[InitialSeg]`
```

Comment:
This note still applies, I'm pretty sure

Reply:
Is it still useful though? As I understand, the notation should be scoped even without this.

Reply:
What makes you think the notation is currently scoped?

Reply:
I recall having to open the namespace to have this notation work. If it's not scoped, should it be?

Reply:
It doesn't seem scoped to me at all. I don't know whether it should be, but certainly it's out of scope (🥁) to scope it or to remove the porting in this PR